schema/schema.cc

```diff
@@ -786,13 +786,8 @@ static bool is_index(replica::database& db, const table_id& id, const schema& s)
     return  db.find_column_family(id).get_index_manager().is_index(s);
 }
 
-static bool is_update_synchronously_view(const schema& s) {
-    auto tag_opt = db::find_tag(s, db::SYNCHRONOUS_VIEW_UPDATES_TAG_KEY);
-    if (!tag_opt.has_value()) {
-        return false;
-    }
-
-    return *tag_opt == "true";
+static std::optional<std::string> is_update_synchronously_view(const schema& s) {
+    return db::find_tag(s, db::SYNCHRONOUS_VIEW_UPDATES_TAG_KEY);
```

Comment:
I think that open-coding this in the call site might be clearer.
If you still want to wrap it, I'd rename the function accordingly, e.g. `get_update_synchronously_view_opt`.